pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -216,8 +216,11 @@ func makeSegmentstorePodSpec(p *api.PravegaCluster) corev1.PodSpec {
 					// is configured with external enabled as they need to wait for the allocation
 					// of the external IP address.
 					// This config gives it up to 5 minutes to become ready.
```

Comment:
Can we remove this line since we are no more setting the values to default

Reply:
Done